packages/modules/babel.config.json

```diff
@@ -1,8 +1,8 @@
 {
     "presets": [
         ["@babel/preset-react",
-        { "runtime": "automatic", "importSource": "@emotion/react" }
-    ],
+            { "runtime": "automatic", "importSource": "@emotion/react" }
+        ],
```

Comment:
looks like this is just a formatting change?

Reply:
👍